build.sbt

```diff
@@ -1,13 +1,13 @@
 import Dependencies._
 
-name := "uncertainty-correlation-study"
+name := "multivariate-prediction-intervals"
 version := "0.1"
 scalaVersion := "2.13.4"
 
-// This commit corresponds to the feature/uncertainty-correlation-archived branch as of 2021-02-17
-val loloVersion = "ab296d0"
+// This commit corresponds to the feature/uncertainty-correlation-archived branch as of 2022-07-28
+val loloVersion = "911e078"
 
 lazy val root = (project in file(".")).dependsOn(loloProject)
-lazy val loloProject = RootProject(uri("git://github.com/CitrineInformatics/lolo.git#%s".format(loloVersion)))
+lazy val loloProject = RootProject(uri("https://github.com/CitrineInformatics/lolo.git#%s".format(loloVersion)))
```

Comment:
`git clone` for Lolo was not working using the git protocol, but seems to be working using the https protocol. Don't know why.